src/services/constants.ts

```diff
@@ -1,8 +1,11 @@
 import assert from 'assert'
 
-const BACKEND_URL: string = process.env.REACT_APP_BACKEND_URL ?? 'http://localhost:8000'
+const BACKEND_URL: string = 'http://localhost:8000'
 assert(typeof BACKEND_URL === 'string')
 
+const STANDARD_DATE_FORMAT: string = ' dd/MM/yyyy'
+
 export {
-  BACKEND_URL
+  BACKEND_URL,
+  STANDARD_DATE_FORMAT
```

Comment:
Would our code formatter accept trailing comma? That way we avoid 2 lines diff for one line change